pom.xml

```diff
@@ -171,6 +172,11 @@
       <version>${version.mockito-core}</version>
       <scope>test</scope>
     </dependency>
+    <dependency>
+      <groupId>com.fasterxml.jackson.dataformat</groupId>
+      <artifactId>jackson-dataformat-yaml</artifactId>
```

Comment:
Looks like `jackson-dataformat-yaml` project would be deprecated in the [future](https://github.com/FasterXML/jackson-dataformat-yaml/blob/master/README.md).
Please update it to `jackson-dataformats-text`

Reply:
I think the repository is deprecated but the artifact `jackson-dataformats-yaml` is not. The code just moved to https://github.com/FasterXML/jackson-dataformats-text but the GAV stays the same.

https://github.com/FasterXML/jackson-dataformats-text/tree/master/yaml#maven-dependency